packages/wdio-webdriver-mock-service/src/WebDriverMock.ts

```diff
@@ -56,17 +59,17 @@ export default class WebDriverMock {
         }
     }
 
-    get(obj, commandName) {
+    get(obj: any, commandName: string) {
         const { method, endpoint, commandData } = protocolFlattened.get(commandName)
 
-        return (...args) => {
+        return (...args: any[]) => {
             let urlPath = endpoint
             for (const [i, param] of Object.entries(commandData.variables || [])) {
-                urlPath = urlPath.replace(`:${param.name}`, args[i])
+                urlPath = urlPath.replace(`:${param.name}`, args[parseInt(i)])
```

Comment:
@christian-bromann @mgrybyk how can i define type for "param" variable. Getting the below issue for now

```
const param: unknown
Object is of type 'unknown'.ts(2571) 
```

Reply:
@suniljaiswal01 you have to type the `protocolFlattened` map properly to have this typed